node/clients/catalog.ts

```diff
@@ -3,7 +3,12 @@ import { JanusClient } from '@vtex/api'
 
 export default class Catalog extends JanusClient {
   constructor(ctx: IOContext, options?: InstanceOptions) {
-    super(ctx, { ...options })
+    super(ctx, {
+      ...options,
+      headers: {
+        VtexIdClientAutCookie: ctx.authToken,
```

Comment:
The header name is with `client` instead of `Client`. [Docs](https://developers.vtex.com/docs/guides/how-to-connect-with-vtex-core-commerce-apis-using-vtex-io#steps)

Reply:
Although apparently it works with `Client`, I agree it is better to change to the name specified in the documentation.